config/initializers/algoliasearch.rb

```diff
@@ -0,0 +1,2 @@
+AlgoliaSearch.configuration = { application_id: ENV['ALGOLIA_APP_ID'], api_key: ENV['ALGOLIA_WRITE_API_KEY'] }
```

Comment:
I'll leave this up to you, but you might want to make these fetches: 

`ENV.fetch('ALGOLIA_APP_ID')` 

`#fetch` will throw an exception if the var isn't set, which makes debugging easier.  The trade-off is that `#fetch` is much trickier to stub in tests.